From 4bff090459d3ea43d7a31542507cad4dd8e040d8 Mon Sep 17 00:00:00 2001
From: "Alexander F. Lent" <lx@xanderlent.com>
Date: Thu, 4 Jul 2024 11:47:24 -0400
Subject: [PATCH 4/4] Install firmware to /lib/firmware instead of
 /lib/firmware/updates

I think the underlying bug is in dracut, but it's easier to work
around it at this level.

Signed-off-by: Alexander F. Lent <lx@xanderlent.com>
---
 firmware/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/firmware/CMakeLists.txt b/firmware/CMakeLists.txt
index 22d86f0..4080a80 100644
--- a/firmware/CMakeLists.txt
+++ b/firmware/CMakeLists.txt
@@ -14,7 +14,7 @@ target_include_directories(${PROJECT_NAME} INTERFACE include)
 file(GLOB FIRMWARE_BINARIES ${CMAKE_CURRENT_SOURCE_DIR}/bin/*.bin)
 if (FIRMWARE_BINARIES)
   install(FILES ${FIRMWARE_BINARIES}
-    DESTINATION lib/firmware/updates/intel/vpu/
+    DESTINATION lib/firmware/intel/vpu/
     PERMISSIONS OWNER_READ
     COMPONENT fw-npu)
 endif()
-- 
2.45.2

